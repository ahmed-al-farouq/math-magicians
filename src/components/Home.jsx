import React from 'react';
import styled from 'styled-components';

const H2 = styled.h2`
  margin: 50px 0 0 160px;
  font-size: 25px;
  font-weight: 600;
  @media screen and (max-width: 768px) {
    margin-left: 70px;
  }

  @media screen and (max-width: 576px) {
    margin-left: 40px;
  }
`;

const P = styled.p`
  margin: 50px auto;
  width: 80%;
  font-size: 18px;
`;

function Home() {
  return (
    <>
      <H2>
        Welcome to our page!
      </H2>
      <P>
        Lorem ipsum dolor sit amet consectetur, adipisicing elit.
        Accusantium quisquam blanditiis quos ullam perferendis incidunt
        qui pariatur aliquam numquam eum voluptatum distinctio molestiae
        quis tenetur illo ipsum, nam unde ipsam?
        Lorem, ipsum dolor sit amet consectetur adipisicing elit.
        Est quos excepturi obcaecati facere porro minus nesciunt,
        ipsam unde temporibus sint sunt blanditiis nihil, qui omnis
        molestiae ab debitis quasi nemo.
      </P>
      <P>
        Lorem ipsum dolor, sit amet consectetur adipisicing elit.
        Saepe itaque modi minima eum rem incidunt, odio placeat
        soluta cum minus nostrum dicta similique quaerat vitae at
        nisi perspiciatis. Fugiat, quaerat!
        Lorem ipsum dolor sit amet consectetur adipisicing elit.
        Consequatur accusamus numquam odit deleniti ab totam fugit
        placeat cum unde quis repellendus corrupti vel, impedit, ex,
        eos magni beatae nobis accusantium.
      </P>
    </>
  );
}

export default Home;
