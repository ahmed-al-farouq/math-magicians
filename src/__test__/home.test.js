import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

it('Match home snapshoot', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
