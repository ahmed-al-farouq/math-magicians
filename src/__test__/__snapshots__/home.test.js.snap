// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match Home snapshoot 1`] = `
Array [
  <h2
    className="sc-bdvvtL jdmmug"
  >
    Welcome to our page!
  </h2>,
  <p
    className="sc-gsDKAQ gzoIuk"
  >
    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Accusantium quisquam blanditiis quos ullam perferendis incidunt qui pariatur aliquam numquam eum voluptatum distinctio molestiae quis tenetur illo ipsum, nam unde ipsam? Lorem, ipsum dolor sit amet consectetur adipisicing elit. Est quos excepturi obcaecati facere porro minus nesciunt, ipsam unde temporibus sint sunt blanditiis nihil, qui omnis molestiae ab debitis quasi nemo.
  </p>,
  <p
    className="sc-gsDKAQ gzoIuk"
  >
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Saepe itaque modi minima eum rem incidunt, odio placeat soluta cum minus nostrum dicta similique quaerat vitae at nisi perspiciatis. Fugiat, quaerat! Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequatur accusamus numquam odit deleniti ab totam fugit placeat cum unde quis repellendus corrupti vel, impedit, ex, eos magni beatae nobis accusantium.
  </p>,
]
`;

exports[`Match home snapshoot 1`] = `
Array [
  <h2
    className="sc-bdvvtL jdmmug"
  >
    Welcome to our page!
  </h2>,
  <p
    className="sc-gsDKAQ gzoIuk"
  >
    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Accusantium quisquam blanditiis quos ullam perferendis incidunt qui pariatur aliquam numquam eum voluptatum distinctio molestiae quis tenetur illo ipsum, nam unde ipsam? Lorem, ipsum dolor sit amet consectetur adipisicing elit. Est quos excepturi obcaecati facere porro minus nesciunt, ipsam unde temporibus sint sunt blanditiis nihil, qui omnis molestiae ab debitis quasi nemo.
  </p>,
  <p
    className="sc-gsDKAQ gzoIuk"
  >
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Saepe itaque modi minima eum rem incidunt, odio placeat soluta cum minus nostrum dicta similique quaerat vitae at nisi perspiciatis. Fugiat, quaerat! Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequatur accusamus numquam odit deleniti ab totam fugit placeat cum unde quis repellendus corrupti vel, impedit, ex, eos magni beatae nobis accusantium.
  </p>,
]
`;
